src/drc/api/serializers.py

```diff
@@ -349,6 +349,12 @@ def validate(self, attrs):
                 _("Lock id is not correct"),
                 code='incorrect-lock-id'
             )
+
+        if self.instance.canonical.latest_version.status == Statussen.definitief:
+            raise exceptions.PermissionDenied(_(
```

Comment:
please change this into a `serializers.ValidationError` with an appropriate code, so that the consumer sees an HTTP 400 instead of a 403. This is not a permission problem, since the token/credentials do give access to the resource. There is also no way around this (403 gives the impression that it would be possible with other rights).